### ISSUE TITLE
Remote SSH does not authenticate properly.

### ISSUE THREAD

**Issue by ESKYoung** [microsoft/vscode-pull-request-github#3098]
- Extension version: v0.32.0
- VSCode Version: 1.62.1
- OS: Darwin x64 21.1.0. VM Instance: Linux 4.19.0-18-cloud-amd64 SMP Debian 4.19.208-1 (2021-09-29) x86_64 GNU/Linux

Steps to Reproduce:

1. Add extension to VS Code
2. Click "Sign into GitHub" and follow all prompts to authenticate
3. SSH into remote VM instance using Remote - SSH
4. Clone any private repository

Expected:

Signing in should give me authentication to clone private repositories.

Actual:

Had to revoke previous GitHub-VS Code authentication, so was setting up again. Running `git clone {HTTPS_REPO_URL}` after the above steps just hangs with no error message. Running `ps -eaf` shows:

```bash
/bin/sh /home/{USER}/.vscode-server/bin/f4af3cbf5a99787542e2a30fe1fd37cd644cc31f/extensions/git/dist/askpass.sh Username for 'https://github.com': 
```

implying that the authentication was not completed successfully, but also no username prompt is shown by VS Code.

Can successfully clone any public repository, or clone private ones if I provide a Personal Access Token in the GitHub URL (`git clone https://{TOKEN}@{REPO_URL}`.

**Comment by ESKYoung** [comment]
Fixed the problem so closing this. For others, [there seems to be a bug in VS Code 1.62.1](https://github.com/microsoft/vscode/issues/136837).

tl;dr: downgrade to 1.62 and switch off auto updating of VS Code for now until 1.62.2 is released!
